Jenkinsfile

```diff
@@ -4,6 +4,12 @@ library 'pipeline-library'
 // Keep logs/reports/etc of last 30 builds, only keep build artifacts of last 3 builds
 properties([buildDiscarder(logRotator(numToKeepStr: '30', artifactNumToKeepStr: '3'))])
 
+// Copy Android SDK/NDK directory paths from Jenkins' environment variables to Google's environment variables.
+// Allows us to run gradle without having to create a "local.properties" file with these paths.
+env.ANDROID_SDK_ROOT = env.ANDROID_SDK
```

Comment:
This likely does not work as you might expect. You're running this outside of the context of any node, so I think it'll basically get/set values from the master Jenkins build machine (while the actual build/test will run on different nodes with likely different paths for the SDK/NDK!)
I'd suggest retaining passing the values in as command line arguments at the build/clean command time.